### PR TITLE
Digitech partially added ergo merge me senpai

### PR DESCRIPTION
**Pull request by ActionHOSchT**
watch out for package-lock.json for $reasons it's not "included"

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,17090 +1,8 @@
 {
   "name": "amct",
   "version": "0.0.0",
-  "lockfileVersion": 2,
```
Comment (binsky08) [inline package-lock.json]: please do not change this lock file

File: src/assets/config.json [hunk 2]
```diff
@@ -30,6 +30,12 @@
       "questions": "assets/tests/soeng1.json",
       "numberOfRandomQuestions": 5,
       "uniqueIdentifier": "soeng1"
+    },
+    {
+      "title": "Digitaltechnik und Rechnersysteme(WiSe 21/22)",
```
Comment (binsky08) [inline src/assets/config.json]: could you please add a whitespace after "Rechnersysteme"

File: src/assets/tests/digitech.json [hunk 3]
```diff
@@ -0,0 +1,623 @@
+[
+  {
+    "name": "Wie nennt sich die für variable Codelänge gängige Codierung, bei der ein Baum gebildet wird?",
+    "type": "radio",
+    "answer": [
```
Comment (binsky08) [inline src/assets/tests/digitech.json]: it should be "answers" (take a look at the README example)
please fix this for any question

File: src/assets/tests/digitech.json [hunk 4]
```diff
@@ -0,0 +1,623 @@
...
+        "value": false
+      },
+      {
+        "text": "Für jede Eingangskombination exakt einen Ausgang schalten. (one hot code)",
+        "value": false
+      }
+    ]
+  },
+  {
+    "name": "Was \"macht\" ein Multiplexer?",
```
Comment (binsky08) [inline src/assets/tests/digitech.json]: is this a duplicate? please remove then

File: src/assets/tests/digitech.json [hunk 5]
```diff
@@ -0,0 +1,623 @@
...
+      },
+      {
+        "text": "... Erstellung einer Wahrheitstabelle.",
+        "value": true
+      }
+    ]
+  },
+  {
+    "name": "In welchen Größen darf man Primimplikanten ermitteln? (x^n bedeutet: x hoch n)",
+    "type": "radio",
```
Comment (binsky08) [inline src/assets/tests/digitech.json]: this can't be a radio type, maybe checkbox?

File: src/assets/config.json [hunk 6]
```diff
@@ -30,6 +30,12 @@
       "questions": "assets/tests/soeng1.json",
       "numberOfRandomQuestions": 5,
       "uniqueIdentifier": "soeng1"
+    },
+    {
+      "title": "Digitaltechnik und Rechnersysteme (WiSe 21/22)",
+      "questions": "assets/tests/digitech.json",
```
Comment (binsky08) [inline src/assets/config.json]: please rename it to digitech1.json to follow the file name scheme

File: src/assets/tests/digitech.json (hunk 3 above)
Comment (ActionHOSchT) [reply src/assets/tests/digitech.json]: my bad, thank you for advice

File: src/assets/tests/digitech.json (hunk 4 above)
Comment (ActionHOSchT) [reply src/assets/tests/digitech.json]: was a duplicate with typo. got removed.

File: src/assets/tests/digitech.json (hunk 5 above)
Comment (ActionHOSchT) [reply src/assets/tests/digitech.json]: my bad, only thought about x^n. is checkbox now.

**Pull request merged**